K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs

```diff
@@ -72,6 +73,16 @@ public async Task GetFieldCaps_WithValidIndex_ReturnFieldCaps()
             mockQueryExecutor.Setup(exec => exec.ExecuteControlCommandAsync(It.IsNotNull<string>(), It.IsAny<RequestContext>()))
                 .Returns(Task.FromResult(testReader));
 
+            using IDataReader dynamicResultReader = new DataReaderMock(new List<Dictionary<string, object>>() {
+                new Dictionary<string, object>() {
+                    { "result", JToken.Parse(@"{""a"": ""int"", ""b"": ""string"", ""c"": {""d"": {""e"": ""string""}}}") },
+                },
+            });
+
+            var calls = new List<QueryData>();
```

Comment:
why did you name this var 'calls'?

Reply:
Because this variable captures the parameter in all of the calls to the method

Reply:
ok got it